codespell_lib/data/dictionary.txt

```diff
@@ -28140,6 +28164,8 @@ untoched->untouched
 untqueue->unqueue
 untrached->untracked
 untranslateable->untranslatable
+untrasform->untransform
```

Comment:
Whilst `untransformed` seems a proper adjective, I'm not convinced that `untransform` is a proper verb? However it seems quite common on grep.app so maybe it should be added to the code dictionary? :shrug:

Reply:
> However it seems quite common on grep.app so maybe it should be added to the code dictionary? :shrug:

The code dictionary is just for typos you **DON'T** want replacing within code, for example `uint->unit` would play havoc with unsigned integers.

I'd probably agree with you @lurch , the verb isn't directional like melt for example. If we can think of a better option say `reverse transform`, we should probably add that as a suggestion too, but probably leave `untransform` in as the clear correction of this typo, i.e. when someone typed `untrasform` they clearly meant to type `untransform` even if grammatically they should have used something else.

We don't currently have a dictionary where the correction is questionable/rare!

Reply:
All good points. Perhaps "undo transform" is a better suggestion for "untransform" than "reverse transform" ?

Reply:
clarification, how should the final entry look like?  
`untrasform->untransform, undo transform,`

Reply:
Yes probably @luzpaz , you could also suggest `reverse transform` too, given the other suggestions are getting into grammar and it probably depends on the context of the sentence.

I think this is the second word we've had recently where the correction is questionable, so perhaps its time to add a dictionary for those...

Reply:
See 00e50e7

Reply:
Thanks @luzpaz , sorry it was just for untransform though, not untransformed so if you can remove the extra suggestions on untransformed please.

Reply:
push